### DISCUSSION TITLE
GitHub disables unnecessary workflows after 60 days if the repository does not have any changes

### DISCUSSION THREAD

**Discussion by toshikijp** [andatoshiki/toshiki-discuss#17] [Q&A] [answered]
GitHub disables unnecessary workflows after 60 days if the repository does not have any changes

**Comment by andatoshiki** [comment, marked as answer]
Warning: To prevent unnecessary workflow runs, scheduled workflows may be disabled automatically. When a public repository is forked, scheduled workflows are disabled by default. In a public repository, scheduled workflows are automatically disabled when no repository activity has occurred in 60 days.

refer to [@GitHub docs](https://docs.github.com/en/actions/managing-workflow-runs/disabling-and-enabling-a-workflow)
